influxql/engine.go

```diff
@@ -130,11 +144,8 @@ func (p *Planner) planExpr(e *Executor, expr Expr) (Processor, error) {
 
 // planCall generates a processor for a function call.
 func (p *Planner) planRawQuery(e *Executor, v *VarRef) (Processor, error) {
-	// Convert the statement to a simplified substatement for the single field.
-	stmt, err := e.stmt.Substatement(v)
-	if err != nil {
-		return nil, err
-	}
+	stmt := e.stmt
```

Comment:
No, because for raw queries there's only a single map function. You can get 1, more, or all fields in a raw query.